pkg/vault/contracts/authorizer/TimelockAuthorizer.sol

```diff
@@ -596,7 +596,26 @@ contract TimelockAuthorizer is IAuthorizer, ReentrancyGuard {
     }
 
     /**
-     * @notice Schedules an execution to set the delay for `actionId`' to `newDelay`.
+     * @notice Schedules an execution to set the delay for `actionId`' to `newDelay`. This makes it impossible to
+     * execute `actionId` without an immutable public on-chain committment for the execution at least `newDelay` seconds
+     * in advance.
+     *
+     * Critical actions that are expected to be performed by EOAs or multisigs are typically subject to said delays to
+     * allow for public scrutiny.
+     *
+     * How long it will take to make this change will depend on the current and new delays: if increasing by more than
+     * 5 days, then the time difference between the delays must pass. Otherwise, the minimum delay change execution
+     * delay of 5 days must pass instead.
+     *
+     * Only `executors` will be able to execute the scheduled action, unless `executors` is an empty array, in which
+     * case any account can execute it.
+     *
+     * Avoid scheduling multiple delay changes for the same action at the same time, as this makes it harder to reason
+     * about the state of the system. If there is already a scheduled delay change and there is a desire to change the
+     * future delay to some other value, cancel the first scheduled change and schedule a new one.
+     *
+     * Only root can call this function, but other accounts may be granted permission to cancel the scheduled execution
+     * (including global cancelers).
```

Comment:
```suggestion
     * @notice Schedules an execution to set the delay for `actionId`' to `newDelay`. This makes it impossible to
     * execute `actionId` without an immutable public on-chain commitment for the execution at least `newDelay` seconds
     * in advance.
     *
     * Critical actions that are expected to be performed by EOAs or multisigs are typically subject to such delays to
     * allow for public scrutiny.
     *
     * How long it will take to make this change will depend on the current and new delays: if increasing by more than
     * 5 days, then the time difference between the delays must pass. Otherwise, the minimum delay change execution
     * delay of 5 days must pass instead.
     *
     * Only `executors` will be able to execute the scheduled action, unless `executors` is an empty array, in which
     * case any account can execute it.
     *
     * Avoid scheduling multiple delay changes for the same action at the same time, as this makes it harder to reason
     * about the state of the system. If there is already a scheduled delay change and there is a desire to change the
     * future delay to some other value, cancel the first scheduled change and schedule a new one.
     *
     * Only root can call this function, but other accounts may be granted permission to cancel the scheduled execution
     * (including global cancelers).
```